src/gui/SignalEstimator.cpp

```diff
@@ -134,4 +145,38 @@ std::optional<std::tuple<QPointF, PointType>> SignalEstimator::parse_(QString bu
     return {};
 }
 
+
+
+std::optional<std::array<double, 3>> SignalEstimator::parseLatency_(QString buffer)
```

Comment:
```C++
std::optional<std::array<double, 3>> SignalEstimator::parseLatency_(QString buffer);
```
My personal opinion on this function declaration, but feel free to ignore it:

- This is a pure function without any side effects, it even does not touch any class state. Its therefore an ideal candidate for a free function instead of a member function. This would make it easier to write unit tests and also increase the cohesion of this class (see also Scott Meyers take on free functions vs. member functions in his _Effective C++_ book).
-  I think that `std::array<double, 3>` is missing an abstraction, I would have chosen a struct with named member variables.
- nitpick: `const QString &` rather than `QString` as the parameter type.